### PR TITLE
Added user check for generate token endpoint.

### PR DESCRIPTION
**Pull request by jrmccannon**
We needed to validate that the user exists before issuing a token for sign-in. This will check Aliases and Credentials.

### CODE REVIEW

File: src/Api/Endpoints/Signin.cs
Comment (Tyrrrz) [inline src/Api/Endpoints/Signin.cs:16]: Nit

Seems like we're slightly inconsistent on the casing of `Signin`/`SignIn`. I thought, when used as a noun, `Signin` is more common. But either way, we should probably stick to one convention.

File: src/Service/Models/SignInTokenRequest.cs
Comment (Tyrrrz) [inline src/Service/Models/SignInTokenRequest.cs:3]: Should this be a record maybe?

**Review by Tyrrrz — APPROVED** [review]

File: src/Service/Models/SignInTokenRequest.cs
Comment (jrmccannon) [reply src/Service/Models/SignInTokenRequest.cs:3]: I added `RequestBase` which is a class.  `RequestBase` has `Origin` and `RpId` which is on the other request endpoints.  Since RequestBase is a class, the request object has to be a class. That was the reason I switched it over.

Comment (Tyrrrz) [reply src/Service/Models/SignInTokenRequest.cs]: Can `RequestBase` be a record then too? 🤔 

This is me nitpicking though

Comment (jrmccannon) [reply src/Service/Models/SignInTokenRequest.cs]: No worries about the nits. Glad we're talking it through. If we change the other request objects to records too, we can change request base to a record. Let's hold off though until this is merged in and then make that a separate pr.

**Pull request merged**
